ruby/2.3.5/Dockerfile

```diff
@@ -97,6 +97,7 @@ RUN set -ex \
 	\
 	&& gem update --system "$RUBYGEMS_VERSION" \
 	&& gem install bundler --version "$BUNDLER_VERSION" --force \
+  && gem install bundler \
```

Comment:
I recommend explicitly specifying the version of the newer `bundler` in these installs.